gym/wrappers/test_frame_stack.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+import numpy as np
+
+import gym
+from gym.wrappers import FrameStack
+
+
+@pytest.mark.parametrize('env_id', ['CartPole-v1', 'Pendulum-v0', 'Pong-v0'])
+@pytest.mark.parametrize('num_stack', [2, 3, 4])
```

Comment:
please make test for Pong conditional on whether atari-py is present (otherwise, will fail, for instance, on python 2.7 tests that don't have atari-py installed)